libs/ui/src/lib/steps/steps/steps.component.stories.ts

```diff
@@ -105,6 +106,13 @@ export const DefaultWithSelectedStep = {
   },
 };
 
+export const DefaultWithScroll = {
+  ...Default,
+  args: {
```

Comment:
When the story is viewed on a screen big enough, the scroll buttons do not appear. Add a `viewport: { defaultViewport: 'tablet' },` parameter to the story (see [the list of viewport ids](https://storybook.js.org/docs/essentials/viewport)) to have the viewport width shrunk enough for a user to see the scroll buttons right away.

Reply:
Yes, I tested with bigger screen and then scrollbar fits, therefore no scroll buttons showed. 'tablet' didn't work, so I used 'ipad'. Maybe because of versions.